src/quo2/components/messages/author/view.cljs

```diff
@@ -0,0 +1,65 @@
+(ns quo2.components.messages.author.view
+  (:require
+   [quo2.components.messages.author.style :as style]
+   [quo.react-native :as rn]
+   [quo2.components.markdown.text :as text]
+   [quo2.components.icon :as icons]
+   [status-im.utils.utils :as utils]))
```

Comment:
status-im shouldn't be used anymore, we have utils in the root level

Reply:
I guess I need to update the guidelines' PR, because in the doc `utils.cljs` is inside `status-im/common/`.